client/src/components/adminLeaderboard.tsx

```diff
@@ -19,6 +20,11 @@ const AdminLeaderboard = ({
   const deleteUser = DeleteUser();
   const softDeleteUser = SoftDeleteUser();
   const updateUser = UpdateUser();
+
+  const insertChanges = (id: string) => {
+    setChangeId([...changeId, id]);
+  };
```

Comment:
Since this function is only called once, and also the content is just one line, you could just call `setChangeId([...changeId, id])` on line 43

Reply:
fixed :)